x-pack/plugins/cases/public/components/custom_fields/list/components/options_field.tsx

```diff
@@ -0,0 +1,206 @@
+      error={errorMessage}
+      isInvalid={isInvalid}
+      fullWidth
+      describedByIds={idAria ? [idAria] : undefined}
+      label={field.label}
+      {...rest}
+    >
+      <EuiPanel color="subdued" paddingSize="s">
+        <EuiDragDropContext onDragEnd={onDragEnd}>
+          <EuiDroppable droppableId="OPTIONS_DROPPABLE_AREA" spacing="m">
+            {currentOptions.map((option, index) => (
+              <EuiDraggable
+                spacing="m"
+                key={`option-${option.key}`}
+                draggableId={`option-${option.key}`}
+                index={index}
+                customDragHandle
+                hasInteractiveChildren
+              >
+                {(provided) => (
+                  <EuiPanel key={index} paddingSize="s">
+                    <EuiFlexGroup alignItems="center" gutterSize="s">
+                      <EuiFlexItem grow={false}>
+                        <EuiPanel
+                          color="transparent"
+                          paddingSize="s"
+                          {...provided.dragHandleProps}
+                          aria-label="Drag Handle"
+                        >
+                          <EuiIcon type="grab" />
+                        </EuiPanel>
+                      </EuiFlexItem>
+                      <EuiFlexItem>
+                        <EuiFieldText
+                          autoFocus={option.isFresh}
+                          fullWidth
+                          value={option.label as string}
+                          placeholder={i18n.LIST_OPTION_PLACEHOLDER_TEXT}
+                          onChange={(e) =>
+                            onChangeOptionLabel({ key: option.key, label: e.target.value })
+                          }
+                          onBlur={() => onBlurOption(option)}
+                          onKeyDown={(e) => {
+                            if (e.key === 'Enter') {
+                              onBlurOption(option);
+                              onAddOption();
+                            }
+                          }}
+                          data-test-subj={`options-field-option-label-${index}`}
+                        />
+                      </EuiFlexItem>
+                      {currentOptions.length > 1 && (
+                        <EuiFlexItem grow={false}>
+                          <EuiButtonEmpty
+                            iconType={'minusInCircle'}
+                            color={'danger'}
+                            onClick={() => onRemoveOption(option.key)}
+                            data-test-subj={`options-field-remove-option-${index}`}
+                          />
+                        </EuiFlexItem>
+                      )}
+                    </EuiFlexGroup>
+                  </EuiPanel>
+                )}
+              </EuiDraggable>
+            ))}
+          </EuiDroppable>
+        </EuiDragDropContext>
+        {maxOptions && currentOptions.length < maxOptions && (
```

Comment:
should we test this?

Reply:
Yes probably, just added this feature and neglected to test, fixing now